docs/docs/how-tos/nebari-local.md

```diff
@@ -73,6 +73,11 @@ The following steps assume you have:
 Note You will be prompted to enter values for some of the choices above if they are absent from the command line arguments (for example, project name and domain)
 :::
 
+:::note
+If you are deploying locally, DNS is not necessary except for dask dashboard and the notebook
+scheduler extension. Therefore for test deployments it is recommended leave out the `--domain` option.
+:::
```

Comment:
```suggestion
:::note
If you are deploying for testing purposes, omitting the `--domain` option is recommended to prevent issues related to setting up a DNS domain for your deployment. Nebari will return the external IP associated with the Kubernetes cluster's load balancer by default.
:::
```